device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/twin/DeviceCapabilities.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.device.twin;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class DeviceCapabilities implements Serializable
+{
+    private static final String IOT_EDGE_NAME = "iotEdge";
+    @Expose
+    @SerializedName(IOT_EDGE_NAME)
+    private Boolean iotEdge = false;
+
+    /**
+     * Setter for iotEdge Boolean
+     *
+     * @param iotEdge - status of capabilities enabled on the device
+     * @throws IllegalArgumentException if the provided argument is null
+     */
+    public void setIotEdge(Boolean iotEdge) throws IllegalArgumentException
```

Comment:
Does this make sense from a device API?

Reply:
There are several other whole classes in this twin package that make no sense to expose on the device side, so I'll remove them all as well.